src/displayapp/screens/settings/SettingSetWakeTime.cpp

```diff
@@ -0,0 +1,103 @@
+#include "displayapp/screens/settings/SettingSetWakeTime.h"
+#include <lvgl/lvgl.h>
+#include <nrf_log.h>
+#include "displayapp/DisplayApp.h"
+#include "displayapp/screens/Symbols.h"
+#include "components/settings/Settings.h"
+#include "displayapp/InfiniTimeTheme.h"
+
+using namespace Pinetime::Applications::Screens;
+
+namespace {
+  constexpr int16_t POS_Y_TEXT = -7;
+
+  void SetTimeEventHandler(lv_obj_t* obj, lv_event_t event) {
+    auto* screen = static_cast<SettingSetWakeTime*>(obj->user_data);
+    if (event == LV_EVENT_CLICKED) {
+      screen->SetTime();
+    }
+  }
+
+  void ValueChangedHandler(void* userData) {
+    auto* screen = static_cast<SettingSetWakeTime*>(userData);
+    screen->UpdateScreen();
+  }
+}
+
+SettingSetWakeTime::SettingSetWakeTime(Pinetime::Controllers::DateTime& dateTimeController,
+                               Pinetime::Controllers::Settings& settingsController,
+                               Pinetime::Applications::Screens::SettingSetSleepWakeTime& settingSetSleepWakeTime)
+  : dateTimeController {dateTimeController}, settingsController {settingsController}, settingSetSleepWakeTime {settingSetSleepWakeTime} {
+
+  uint8_t h, m;
+
+  lv_obj_t* title = lv_label_create(lv_scr_act(), nullptr);
+  lv_label_set_text_static(title, "Set current time");
```

Comment:
I believe you said you changed this? maybe you forgot to push the change?
```suggestion
  lv_label_set_text_static(title, "Set wake time");
```

Reply:
I must've only changed it in my InfiniSim checkout.  Will update, along with all your suggestions.